docs/developer/end-to-end-tests.md

```diff
@@ -92,10 +92,10 @@ Our CI system relies on the [kubeapps/integration-tests](https://hub.docker.com/
 The `kubeapps/integration-tests` image is built using this [Makefile](../../integration/Makefile). Manually edit this file to specify the proper version tag.
 
 ```bash
+# Get the tags from https://hub.docker.com/r/kubeapps/integration-tests
```

Comment:
Could link directly to https://hub.docker.com/r/kubeapps/integration-tests/tags?page=1&ordering=last_updated . But more importantly, although it should be obvious, you don't mention here to "and then increment the patch version of the latest tag to get the `IMAGE_TAG` that you'll use below.